### PR TITLE
Math magicians: events

### PR DESCRIPTION
**Pull request by Goruchie**
# **Implement the following requirements:**
- **Copy the files calculate.js and operate.js into a logic/ directory in your project.**
- **Import the files in your Calculator component.**
- **Implement the event handlers you need to use the math logic from calculate.js and operate.js in the React component.**
- **Test the app in the browser. It should be able to perform math operations.**

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,40 +1,78 @@
...
   }
 
-  render() {
-    return (
-      <div className="container">
-        <div className="keypad">
-          <input className="div1" type="number" placeholder="0" />
-          <button type="button">AC</button>
-          <button type="button">+/-</button>
-          <button type="button">%</button>
```
Comment (peterrobert) [inline src/components/Calculator.js:17]: - [ ] It seems these two buttons are not functioning as expected. For example, the `+/-` button should make the value negative or positive. As per your implementation, the value doesn't change at all. Kindly fix this.

Also, the `%`  button should work as modulus. As per your implementation, there is no visible change to the value when you click this button. Kindly fix this too.

**Review by peterrobert — CHANGES_REQUESTED** [review]
Hi @Goruchie, 

Very nice project. I love the way you have separated your code into modules and assigned specific tasks to specific modules 👏🏿 . However, there are a few changes to be made kindly check below ⬇️

### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (Goruchie) [reply src/components/Calculator.js:17]: @peterrobert Hello! thank you for the review, I fixed the '+/-' button but the '%' it's impossible for me, I don't know how to fix it, and I saw a lot of people which have approved this project without that button working and they are not able to help me, can you help me, please?

**Review by codecaiine — APPROVED** [review]
Hi @Goruchie  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
You've followed the Gitflow rules. :heavy_check_mark:
You have good commit messages :heavy_check_mark:
No linter errors :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**